### PR TITLE
new design implementation for footer and removed the pre-footer section

### LINKED ISSUE #6035: Implement the new design for Footer and pre-footer section .
Screenshot 2024-07-01 at 5 54 15 PM

Proposed Design :

Screenshot 2024-07-01 at 5 54 48 PM

### PR DESCRIPTION
**Pull request by ShravaniAK**
resolves #6035 

changes made :
1. replaced the footer background colour.
2. Move footer icons to the right.
3. Remove the pre-footer section of the previous design.
4. added space at the top of the cncf logo container.

proposed design :
Screenshot 2024-07-01 at 6 30 29 PM

Implementaion:

Screenshot 2024-07-01 at 6 30 45 PM

### CODE REVIEW

File: docs/stylesheets/extra.css [hunk 1]
```diff
@@ -358,6 +359,12 @@ border-radius: 20% 0 0 0 ;
   background-color: var(--md-footer-bg-color);
 }
 
+.md-footer-meta__inner{
+  display:flex;
+  flex-direction: row;
+  margin:0.5rem 1.5rem;
```
Comment (Cali0707) [inline docs/stylesheets/extra.css:365]: nit:
```suggestion
  display: flex;
  flex-direction: row;
  margin: 0.5rem 1.5rem;
```

File: docs/stylesheets/extra.css [hunk 2]
```diff
@@ -3,6 +3,7 @@
   --md-primary-fg-color--dark: #0865AD;
   --md-primary-fg-color--light: #6695CA;
   --nav-item-color: #4d4d4d;
+  --md-footer-bg-color:#081729;
```
Comment (Cali0707) [inline docs/stylesheets/extra.css:6]: nit:
```suggestion
  --md-footer-bg-color: #081729;
```

**Review by Cali0707 — CHANGES_REQUESTED** [review]

**Comment by ShravaniAK** [comment]
@Cali0707 I have fixed the spacing issue , thanks !

**Review by Cali0707 — CHANGES_REQUESTED** [review]
Nice work @ShravaniAK - this looks a lot better now! I left one code quality comment and then we should be good to merge this

File: docs/stylesheets/extra.css [hunk 3]
```diff
@@ -488,3 +497,10 @@ border-radius: 20% 0 0 0 ;
     color: var(--nav-item-color);
   }
 }
+/* @media screen and (max-width:2600px){
+.md-footer-meta {
+  justify-content: space-between;
+}
+} */
```
[the remark this answers is not in the document]
Comment (ShravaniAK) [reply docs/stylesheets/extra.css]: yes done !

**Comment by ShravaniAK** [comment]
@Cali0707 I have removed the commented code, thanks.

**Review by Cali0707 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**